src/main/java/org/mule/extension/file/internal/command/AbstractLocalCopyCommand.java

```diff
@@ -83,8 +84,8 @@ protected final void execute(String sourcePath, String target, boolean overwrite
     try {
       doExecute(source, targetPath, overwrite, copyOption != null ? new CopyOption[] {copyOption} : new CopyOption[] {});
     } catch (FileAlreadyExistsException e) {
-      throw new org.mule.extension.file.common.api.exceptions.FileAlreadyExistsException(format("Can't copy '%s' to '%s' because the destination path "
-          + "already exists. Consider setting the 'overwrite' parameter to 'true'", source.toAbsolutePath(),
+      throw new org.mule.extension.file.common.api.exceptions.FileAlreadyExistsException(format("Found exception %s '%s' to '%s'. Destination path "
```

Comment:
Nothing in mule needs to refer to the term `exception`. To me the original error was better, the only thing you need to switch is the copy/move word